first_django_pr/my_app/templates/my_app/base.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>$Title$</title>
```

Comment:
In this work expects that you will extend the base template with the using of Django tag {% block <block_name> %} ... {% endblock %}